repo-scripts/size-analysis/bundle-definitions/functions.json

```diff
@@ -30,3 +30,4 @@
     ]
   }
 ]
+
```

Comment:
This file still seems to show up in the diff, maybe just `git checkout master repo-scripts/size-analysis/bundle-definitions/functions.json`?

Reply:
SGTM. Easy enough approach.